engine/src/main/java/org/terasology/rendering/world/WorldRenderer.java

```diff
@@ -23,60 +23,227 @@
 import org.terasology.world.WorldProvider;
 import org.terasology.world.chunks.ChunkProvider;
 
+/**
+ * Implementations of this class are responsible for rendering the whole 3D world,
+ * inclusive of background.
+ *
+ * While 2D GUIs are dealt with elsewhere, it is in the remit of implementations of this class
+ * to provide Augmented-Reality-Style user interfaces, i.e. boxes around selected objects
+ * 3d-positioned labels above player's heads or held-in-hand tools.
+ *
+ * If this is the first time you look into this interface, you should probably look at the
```

Comment:
This paragraph sounds a bit odd because of the "if ths is the first time you're reading this..." false conditional - may need a rework.
I don't recall ever seeing documentation where methods are designated "important", anyway.